src/backend/database_migrations/versions/20230110_175025_remove_czb_failed_genome_recovery_field.py

```diff
@@ -0,0 +1,20 @@
+"""drop the czb failed genome recovery column
+
+Create Date: 2023-01-10 17:50:35.250700
+
+"""
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "20230110_175025"
+down_revision = "20230109_223207"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.drop_column("samples", "czb_failed_genome_recovery", schema="aspen")
```

Comment:
Can we move this migration to a separate PR and deploy it after we've stopped depending on this field?

Reply:
sure